nf_core/download.py

```diff
@@ -270,53 +417,128 @@ def prompt_use_singularity_cachedir(self):
                 if cachedir_path:
                     os.environ["NXF_SINGULARITY_CACHEDIR"] = cachedir_path
 
-                    # Ask if user wants this set in their .bashrc
-                    bashrc_path = os.path.expanduser("~/.bashrc")
-                    if not os.path.isfile(bashrc_path):
-                        bashrc_path = os.path.expanduser("~/.bash_profile")
-                        if not os.path.isfile(bashrc_path):
-                            bashrc_path = False
-                    if bashrc_path:
+                    """
+                    Optionally, create a permanent entry for the NXF_SINGULARITY_CACHEDIR in the terminal profile.
+                    Currently support for bash and zsh.
+                    ToDo: "sh", "bash", "dash", "ash","csh", "tcsh", "ksh", "zsh", "fish", "cmd", "powershell", "pwsh"?
+                    """
+
+                    if os.environ["SHELL"] == "/bin/bash":
```

Comment:
I was testing this on Gitpod, and it failed because there isn't a variable `$SHELL`. We should probably check first if the variable exists.

Reply:
Good catch! I switched it to `os.getenv("SHELL", "")` now, which has a default value if the variable is not set.